doc/ref_arch/kubernetes/chapters/chapter02.md

```diff
@@ -23,7 +28,120 @@
 <a name="2.2"></a>
 ## 2.2 Reference Model Requirements
 
-Traceability to Reference Model.
+This reference model intends to implement both the Basic, Network Intensive and Compute Intensive Instance types defined in the Reference Model. The following table contains the requirements of the Reference Model and indicates if the Reference Architecture supports them.
+
+<a name="2.2.1"></a>
+### 2.2.1 Network Acceleration Extensions
+
+| Attribute | Description | Supported |
+|-----------|----------------------------------|-------|
+| .il-ipsec | In-line IPSec acceleration | optional <sup>1)</sup> |
+| .la-crypto | Look-Aside encryption/decryption engine | optional <sup>2)</sup> |
+| .la-trans | Look-Aside Transcoding acceleration | optional <sup>3)</sup> |
+| .la-programmable | Look-Aside programmable acceleration | optional <sup>4)</sup> |
+
+**1)** In Kubernetes based infrastructures virtio-ipsec can be applied only if the CNI plugin is OVS.<br>
```

Comment:
Instead of 4 the notes can be added to the table entries themselves. Since this chapter is in RA-2, the language can be simplified:

| Attribute | Description | Supported |
|-----------|----------------------------------|-------|
| .il-ipsec | In-line IPSec acceleration | optional and only if the CNI plugin is OVS |
| .la-crypto | Look-Aside encryption/decryption engine | optional and only if the CNI plugin is OVS |
| .la-trans | Look-Aside Transcoding acceleration | optional and only if the CNI plugin is OVS |
| .la-programmable | Look-Aside programmable acceleration | optional and only if the CNI plugin is OVS |